src/Protocol/Request/RequestHeader.php

```diff
@@ -193,30 +200,4 @@ public function processAuthorizationHeader($header)
 
 		return $headers;
 	}
-
-	/**
-	 * Authenticate an identity using HTTP Basic authentication for the Request.
-	 *
-	 * @return  integer  Identity ID for the authenticated identity.
-	 *
-	 * @since   1.0
-	 */
-	protected function doBasicAuthentication()
-	{
-		// If we have basic auth information attempt to authenticate.
-		$username = $this->_input->server->getString('OAUTH_CLIENT_ID');
-
-		if ($username)
-		{
-			$password = $this->_input->server->getString('PHP_AUTH_PW');
-
-			$identityId = $this->doPasswordAuthentication($username, $password);
```

Comment:
This function doesn't exist and `doBasicAuthentication` doesn't seem to be called. So I got rid of all of it